src/routes/api/app.js

```diff
@@ -18,6 +26,14 @@ module.exports = function (express) { // Setting module.exports to a new functio
 
 */
 
+/**
+ * @function get
```

Comment:
Desc should be under the @function not at the bottom also you need to inline the jsdoc annotation with the actual function if your confused look at my branch carsonTest or talk to me  #22